2/scss/active.scss

```diff
@@ -0,0 +1,117 @@
+@import "vars";
+@import "mixins";
+.back-icon {
+  width: 24px;
+  height: 18px;
+  position: relative;
+  border: 0;
+  background-color: transparent;
+  cursor: pointer;
+  &::after {
+    @include style-back-icon;
+    transform: rotate(45deg);
+  }
+  &::before {
+    @include style-back-icon;
+    transform: rotate(135deg);
+  }
+}
+
+.none {
```

Comment:
this is helper class - why it is in `active.scss`? Actually O do not understand what `active` means in this context :)

better name for this class would be `.d-none` or `.hidden`

Reply:
renemed